### DISCUSSION TITLE
3월 14일 회의록 ( 3rd week )

### DISCUSSION THREAD

**Discussion by zangsu** [zangsu/cs-test-service#5] [Announcements]
+ API 명세 초본 수정
+ 주간 스프린트 설정

**Comment by zangsu** [comment]
API 명세 수정을 위한 회의를 진행하겠습니다.

**Reply by zangsu** [reply]
### 참여자
장혁수, 문경덕


### 작업 내용
![image](https://user-images.githubusercontent.com/76612738/224921883-a63fe48f-927d-40ab-8c67-4719fb0aab64.png)

**Comment by Moon-GD** [comment]
@zangsu 프로젝트 시작하다 보니 수정 요구 사항이 발견되었습니다.
+ git branch naming convention 수정
+ Issue Template 내용 수정

다음 회의 때 함께 보완해보면 좋을 것 같아요 😄

**Reply by zangsu** [reply]
확인했습니다. 금일 오후에 회의 진행하겠습니다!
관련 내용 중 참고사항이 있다면 미리 공유 부탁드립니다!

**Reply by zangsu** [reply]
### 참여자
장혁수, 문경덕

### 작업 내용
git wiki 수정
